tests/nested/manual/cloud-init-never-used-not-vuln/task.yaml

```diff
@@ -0,0 +1,151 @@
+summary: |
+    Test that cloud-init is no longer vulnerable on Ubuntu Core with the fix for
+    CVE-2020-11933 in place.
+
+systems: [ubuntu-18.04-64, ubuntu-16.04-64]
+
+environment:
+    # this test ensures that existing images without the fix are no longer
+    # vulnerable after refreshing to a version of snapd with the fix
+    BUILD_FROM_CURRENT/refresh: false
+    BUILD_FROM_CURRENT/firstboot: true
+    USE_CLOUD_INIT: false
+
+prepare: |
+    #shellcheck source=tests/lib/nested.sh
+    . "$TESTSLIB/nested.sh"
+
+    create_nested_core_vm
+    start_nested_core_vm
+
+    # to build the cloud-init NoCloud cdrom drive
+    apt install -yqq genisoimage
+
+    # build the cloud-init NoCloud cdrom drive
+    cd seed
+    genisoimage -output ../seed1.iso -volid cidata -joliet -rock user-data meta-data
+
+restore: |
+    #shellcheck source=tests/lib/nested.sh
+    . "$TESTSLIB/nested.sh"
+    destroy_nested_vm
+    cleanup_nested_env
+
+    apt remove -yqq genisoimage || true
```

Comment:
don't remove it because a following test could need it

Reply:
fixed